experimental/PropertyDDS/packages/property-inspector-table/src/utils.tsx

```diff
@@ -332,9 +336,9 @@ export const getReferenceValue = (rowData: IInspectorRow) => {
 
 const getShortId = (parentPath: string, childId: string | undefined = undefined): string => {
   const sanitizer = [
-    { searchFor: /[.]|[\[]/g, replaceWith: idSeparator },
-    { searchFor: /[\]]/g, replaceWith: '' },
-    { searchFor: /\/[\/]+/g, replaceWith: idSeparator },
+    { searchFor: /[.[]/g, replaceWith: idSeparator },
+    { searchFor: /]/g, replaceWith: "" },
+    { searchFor: /\/\/+/g, replaceWith: idSeparator },
```

Comment:
Note to self: I think these regex simplifications are correct (you don't need to escape ']' when not inside a '[..]')